# Cenários que falharam na evolução
10
643,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:58:52.351,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
63982,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
65,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:20:06.231,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
6452,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
378,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:41:23.234,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
37486,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
78,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:21:23.342,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
7668,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
154,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:26:16.123,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
15173,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
574,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:54:31.675,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
56883,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
432,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:44:49.638,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
42846,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
329,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:37:49.31,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
33061,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
277,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:34:11.94,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
27613,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
557,Entry point for SctpEchoTest.testSimpleEcho,1,2014-04-28 13:52:51.148,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho()
0
1
55411,io.netty.testsuite.transport.sctp.SctpEchoTest.testSimpleEcho(),-1,org.junit.internal.AssumptionViolatedException: got: <false>, expected: is <true>,1.0
0
-----------------------------------
